Android/app/src/main/java/app/intra/sys/IntraVpnService.java

```diff
@@ -224,7 +224,12 @@ private void updateServerConnection() {
     if (equalUrls(url, null)) {
       // Use the Google Resolver
       AssetManager assets = this.getApplicationContext().getAssets();
-      newConnection = GoogleServerConnection.get(new GoogleServerDatabase(this, assets));
+      final GoogleServerConnection googleServerConnection =
+          GoogleServerConnection.get(new GoogleServerDatabase(this, assets), null);
+      if (googleServerConnection != null && googleServerConnection.didBootstrapWithFallback()) {
+        bootstrap.putBoolean(Names.FALLBACK_HOSTNAME.name(), true);
```

Comment:
Let's make the parameter be a string and name it `FALLBACK` with a value `FRONTED` or something like that, so we can later have something like `SPLIT`.

Reply:
Done in 045c3d6.